sdk/monitor/Azure.Monitor.Ingestion/api/Azure.Monitor.Ingestion.netstandard2.0.cs

```diff
@@ -30,7 +30,7 @@ public partial class UploadLogsOptions
         public UploadLogsOptions() { }
         public int MaxConcurrency { get { throw null; } set { } }
         public Azure.Core.Serialization.ObjectSerializer Serializer { get { throw null; } set { } }
-        public event Azure.Core.SyncAsyncEventHandler<Azure.Monitor.Ingestion.UploadFailedEventArgs> UploadFailedEventHandler { add { } remove { } }
+        public event Azure.Core.SyncAsyncEventHandler<Azure.Monitor.Ingestion.UploadFailedEventArgs> UploadLogsFailed { add { } remove { } }
```

Comment:
Why `UploadLogsFailed` when the method is just called `Upload`?  All of these names should be aligned.

Reply:
I think this is because we might have an UploadMetricsFailed EventHandler in the future. @srnagar